### PR TITLE
docs: improvement to semantics

### PR DESCRIPTION
**Pull request by mffap**

### CODE REVIEW

**Comment by mffap** [comment]
@fforootd  a mix of smaller changes to the docs. Also corrected the title heading for management roles.

File: site/docs/administrate/02-organisations.en.md [hunk 1]
```diff
@@ -105,9 +124,14 @@ Congratulations your are done! You can check this by visiting [https://console.z
...
 
-> This only works when the [user](administrate#Users) is member of this [organisation](administrate#Organisations)
+> This only works when the [user](administrate#Users) is member of this [organization](administrate#Organizations)
+
+### Manage Organization ZITADEL Roles
+
+You can assign users [management roles](https://docs.zitadel.ch/administrate#ZITADEL_s_management_Roles) to your new organization.
 
-### Manage Organisation ZITADEL Roles
+[//]: # (@fforootd screenshot sais "projects" - bit confusing)
```
Comment (fforootd) [inline site/docs/administrate/02-organisations.en.md:133]: Managers can be "managed" on the IAM resouces e.g projects

File: site/docs/administrate/02-organisations.en.md [hunk 2]
```diff
@@ -105,9 +124,14 @@ Congratulations your are done! You can check this by visiting [https://console.z
...
+[//]: # (@fforootd Is a "Manager" another "Organisation owner", or is this a special role?)
```
Comment (fforootd) [inline site/docs/administrate/02-organisations.en.md:134]: Org. Owner is a Manager Role 

Generally all IAM roles are Manager Roles

File: site/docs/administrate/03-projects.en.md [hunk 3]
```diff
@@ -6,37 +6,37 @@ title: Projects
...
 
 **Grants**
 
-With ZITADEL it is possible to give third parties (other organisations) the possibility to manage certain roles on their own.
-To achieve this the owner of a project can grant (some could say delegate) certain roles or all roles to an organisation.
-After granting that organisation it can manage on its own which user has what roles.
-This feature is especially useful for service providers, because they are able to establish a great self-service culture for their business customers.
+With ZITADEL it is possible to give third parties (other organisations) the possibility to manage certain roles on their own. As a service provider, you will find this feature useful, as it allows you to establish a self-service culture for your business customers.
+
+The owner of a project can grant (some would say "delegate") certain roles or all roles to another organisation. The target organisaton can then indipendently manage the assignment of their users to  the role within the [granted project](administrate#Project_vs_granted_Project).
```
Comment (fforootd) [inline site/docs/administrate/03-projects.en.md:29]:
```suggestion
The owner of a project can grant (some would say "delegate") certain roles or all roles to another organisation. The target organization can then indipendently manage the assignment of their users to  the role within the [granted project](administrate#Project_vs_granted_Project).
```

File: site/docs/administrate/07-policies.en.md [hunk 4]
```diff
@@ -7,11 +7,13 @@ title: Policies
...
 
+### Available policies
+
 Below is a list of available policies
 
-### Password complexity
+#### Password complexity
 
-This policy enforces passwords of users within the org. to be compliant.
+This policy enforces passwords of users within the organisation to be compliant.
```
Comment (fforootd) [inline site/docs/administrate/07-policies.en.md:16]:
```suggestion
This policy enforces passwords of users within the organization to be compliant.
```

File: site/docs/start/00-quick-start.en.md [hunk 5]
```diff
@@ -5,12 +5,18 @@ description: A quick-start reference for the impatient reader.
 
 > All documentations are under active work and subject to change soon!
 
-### Try ZITADEL
+Hi there,
```
Comment (fforootd) [inline site/docs/start/00-quick-start.en.md:8]: Is this by design 😁

**Review by fforootd — CHANGES_REQUESTED** [review]

File: site/docs/administrate/02-organisations.en.md [hunk 6]
```diff
@@ -1,82 +1,101 @@
...
-or if you think of Google "Gmail" vs "Gsuite".
+The global organization holds users that are not assigned to any other organization in the [System](administrate#What_is_meant_by_system). Thus ZITADEL provides a global organization for users who manage their accounts on their own.
 
-### Create an organisation without existing login
+<details open>
+    <summary>
+        Example
+    </summary>
+Let's look at our example company `acme.ch`: Suppose ACME sells online-tickets for concert venues. ACME created an organization `iam` to manage their own enterprise users (employees) and projects to manage the provided services. They also created an organization `b2b-partner-1`, allowing the partner self-manage their access. A partner could be a concert venue, that can administrate the backend of the service (e.g. posting new concerts, setting up billing, ...), and you want to allow them to self-manage access of users (e.g. employees of the venue) to their backend. Lastly, the organization `global` holds all the b2c customers of `acme.ch` that registered to the service to buy concert tickets.
+</details>
```
Comment (fforootd) [inline site/docs/administrate/02-organisations.en.md:22]: @peintnermax will those work?

**Review by fforootd — APPROVED** [review]

**Pull request merged**